src/app/routers/field.py

```diff
@@ -170,3 +173,45 @@ def get(self):
             response.append(field)
 
         return jsonify(response)
+
+@FIELDS_NS.route("/<int:field_id>")
+class FieldAPI(Resource):
+    """
+        Field/{id} API
+
+        url: '/fields/{id}'
+        methods: GET, PUT, DELETE
+    """
+
+    @API.doc(
+        responses={
+            200: 'OK',
+            403: 'User is not the field owner',
+            404: 'Field not found',
+        }, params={
+            'field_id': 'Field id'
+        }
+    )
+    # pylint: disable=no-self-use
+    def get(self, field_id):
+        """
+        Get field by id
+
+        :param field_id: field id
+        :return: json
+        """
+        field = FieldService.get_by_id(field_id)
+
+        if field is None:
+            raise BadRequest("Field does not exist")
```

Comment:
BadRequest is 400 response. No 400 response in API.doc

Reply:
added